vite.config.ts

```diff
@@ -0,0 +1,37 @@
+import { resolve } from 'path';
+import { defineConfig } from 'vite';
+import vue from '@vitejs/plugin-vue';
+import checker from 'vite-plugin-checker';
+import svgLoader from 'vite-svg-loader';
+import dts from 'vite-plugin-dts';
+import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+	plugins: [
+		vue(),
+		checker({
+			typescript: true,
+		}),
+		svgLoader(),
+		dts(),
+		cssInjectedByJsPlugin(),
```

Comment:
Let's explain in a comment why we need this.

Reply:
Isn't it self explanatory? Webpack used to build in css into js by default, vite requires additional plugin for that. We rely on that - we don't import component styles separately in platform